cmd/fsi.go

```diff
@@ -34,8 +34,8 @@ func NewFSICommand(f Factory, ioStreams ioes.IOStreams) *cobra.Command {
 	}
 
 	unlink := &cobra.Command{
-		Use:   "unlink",
-		Short: "unlink a .qri-ref",
+		Use:   "unlink [DATASET]",
+		Short: "unlink a dataset from a directory on disk",
```

Comment:
While trying to make sure I was documenting correctly, I realized this only takes one arg in practice (even though it parses a second arg, it never uses it). Wondering if we should be setting `Args: cobra.MaximumNArgs(1)` here.